playwright/_impl/_video.py

```diff
@@ -55,6 +55,10 @@ async def path(self) -> pathlib.Path:
         return artifact.absolute_path
 
     async def save_as(self, path: Union[str, pathlib.Path]) -> None:
+        if self._page._connection._is_sync and not self._page._is_closed:
+            raise Error(
+                "Page is not yet closed. Close the page prior to calling save_as"
```

Comment:
```suggestion
                "Page is not yet closed. Close the page prior to calling Video.save_as"
```

Reply:
Left as is—this looks funny since it's not clear what `Video` is in this context. (In this case, just a type.)

Most will be calling it like `page.video.save_as(…)` or `video.save_as(…)`.